joints: [   'shoulder_1_joint',
            'shoulder_2_joint']

shoulder_1_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: shoulder_1_joint
    joint_speed: 0.8
    motor:
        id: 1
        init: 512
        min: 0
        max: 1023

shoulder_2_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: shoulder_2_joint
    joint_speed: 0.8
    motor:
        id: 2
        init: 512
        min: 0
        max: 1023

